FILES/GetTattooShopDlcItemData.md

```diff
@@ -9,51 +9,37 @@ aliases: ["0xFF56381874F82086","_GET_TATTOO_COLLECTION_DATA"]
 BOOL GET_TATTOO_SHOP_DLC_ITEM_DATA(int characterType, int decorationIndex, Any* outComponent);
 ```
 
-Character types:
-```
-0 = Michael,
-1 = Franklin,
-2 = Trevor,
-3 = MPMale,
-4 = MPFemale
-```
+Returns data that adheres to the tattoo shop item data that is used in shop_tattoo.meta
 
-```csharp
-enum TattooZoneData
```

Comment:
Why was this removed?

Reply:
hey so i removed this since the tattoo zones don't have any use in this specific native (like, the struct doesn't return the zone or anything, nor do the parameters need it)